Makefile

```diff
@@ -18,39 +62,39 @@ python-caches-clean:
 	-@find . -name "__pycache__" -exec rm -rf -- \{\} \;
 	-@find . -name ".pyc" -exec rm -rf -- \{\} \;
 
-docs:
+docs: check-venv
 	cd docs && $(MAKE) html
 
-test:
-	python3 setup.py test
+test: check-venv
+	$(VEPYTHON) setup.py test
 
-it: python-caches-clean
-	tox
+it: check-venv python-caches-clean
+	. $(VENV_ACTIVATE_FILE); tox
 
-it34: python-caches-clean
-	tox -e py34
+it34: check-venv python-caches-clean
+	. $(VENV_ACTIVATE_FILE); tox -e py34
 
-it35: python-caches-clean
-	tox -e py35
+it35: check-venv python-caches-clean
+	. $(VENV_ACTIVATE_FILE); tox -e py35
 
-it36: python-caches-clean
-	tox -e py36
+it36: check-venv python-caches-clean
+	. $(VENV_ACTIVATE_FILE); tox -e py36
 
-it37: python-caches-clean
-	tox -e py37
+it37: check-venv python-caches-clean
+	. $(VENV_ACTIVATE_FILE); tox -e py37
 
-benchmark:
-	python3 setup.py pytest --addopts="-s benchmarks"
+benchmark: check-venv
+	$(VEPYTHON) setup.py pytest --addopts="-s benchmarks"
 
-coverage:
-	coverage run setup.py test
-	coverage html
+coverage: check-venv
+	. $(VENV_ACTIVATE_FILE); coverage run setup.py test
+	. $(VENV_ACTIVATE_FILE); coverage html
 
-release-checks:
+release-checks: check-venv
 	./release-checks.sh $(release_version) $(next_version)
```

Comment:
I think we want to prefix this with `. $(VENV_ACTIVATE_FILE); ` to ensure that `python3` used here it the one from venv right?

Reply:
Right! Thanks for catching, fixed.